<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>adicionar_cliente</class>
 <widget class="QDialog" name="adicionar_cliente">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>869</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Adicionar Cliente</string>
  </property>
  <property name="windowIcon">
   <iconset resource="recursos.qrc">
    <normaloff>:/icons/add-3.png</normaloff>:/icons/add-3.png</iconset>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>100</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Adicione um cliente</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>140</y>
     <width>251</width>
     <height>228</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>insira os dados</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>cpf</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="txt_cpf"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>conta</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="txt_conta"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>senha (padrao)</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="txt_senha"/>
    </item>
    <item row="7" column="1">
     <widget class="QPushButton" name="btn_add">
      <property name="text">
       <string>Adicionar</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>nome</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="txt_nome"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="txt_cpf_g"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>seu cpf</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>txt_cpf</tabstop>
  <tabstop>txt_cpf_g</tabstop>
  <tabstop>txt_nome</tabstop>
  <tabstop>txt_conta</tabstop>
  <tabstop>txt_senha</tabstop>
  <tabstop>btn_add</tabstop>
 </tabstops>
 <resources>
  <include location="recursos.qrc"/>
 </resources>
 <connections/>
</ui>
